inference-engine/src/readers/ir_reader/CMakeLists.txt

```diff
@@ -29,6 +29,8 @@ target_compile_definitions(${TARGET_NAME} PRIVATE IMPLEMENT_INFERENCE_ENGINE_PLU
 target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}")
 
 target_link_libraries(${TARGET_NAME} PRIVATE ngraph
+                                             ir_frontend
```

Comment:
BTW, FEM detects FEs by pattern `.*_ngraph_frontend` while this one is named differently. Does it really work?

Reply:
Good question, IR FE was used only directly inside parser without FEM. I'll doublecheck.

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/7249